api/app.js

```diff
@@ -1,17 +1,24 @@
 const express = require('express');
 const app = express();
-const User = require('./models/user.js');
-const userRouter = require('./routes/userRouter.js');
+const userRouter = require('./routes/userRouter');
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
Could dotenv.config(); be executed config.js only and removed from other places?

Reply:
the config was changed to preload dotenv